### PR TITLE
dialect/sql/entsql: support including non-key columns in postgres indexes

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: doc/md/schema-indexes.md [hunk 1]
```diff
@@ -234,6 +234,12 @@ func (User) Indexes() []ent.Index {
                     dialect.Postgres: "GIN",
                 }),
             ),
+		// For PostgreSQL, we can include in the index
+		// non-key columns.
+		index.Fields("workplace").
+			Annotations(
+				entsql.IncludeColumns("address"),
+			),
```
[the remark this answers is not in the document]
Comment (a8m) [reply doc/md/schema-indexes.md:242]: Hey and thanks for the review.

What do you mean by “generated examples”?

[the remark this answers is not in the document]
Comment (thuan1412) [reply doc/md/schema-indexes.md:242]: I mean [this section](https://github.com/ent/ent/blob/17d45c7f234ca4300b918b628967ae083514d43c/doc/md/schema-indexes.md?plain=1#L247-L260) in the doc

[the remark this answers is not in the document]
Comment (a8m) [reply doc/md/schema-indexes.md:242]: Oh, great catch. Would you like to contribute this?

[the remark this answers is not in the document]
Comment (thuan1412) [reply doc/md/schema-indexes.md:242]: Yep. I will create a PR.

[the remark this answers is not in the document]
Comment (a8m) [reply doc/md/schema-indexes.md:242]: Thanks 🙏🏻

**Pull request merged**